src/rosdiscover/interpreter/interpreter.py

```diff
@@ -273,21 +274,31 @@ def _load(self,
 
         try:
             model = self.models.fetch(pkg, nodetype)
+            if isinstance(model, PlaceholderModel) and name != nodetype:
+                model = self.models.fetch(pkg, name)
```

Comment:
might be worth adding a note about why `name` is used here

Reply:
Done